zivid_camera/CMakeLists.txt

```diff
@@ -30,21 +30,9 @@ find_package(catkin REQUIRED COMPONENTS
   nodelet
 )
 
-find_package(Zivid COMPONENTS Core REQUIRED)
+find_package(Zivid 1.6.0 COMPONENTS Core REQUIRED)
 message(STATUS "Found Zivid version ${Zivid_VERSION}")
 
-set(SUPPORTED_ZIVID_MAJOR_VERSION 1)
-set(MIN_SUPPORTED_ZIVID_VERSION 1.3.0)
-
-if((NOT Zivid_VERSION_MAJOR EQUAL SUPPORTED_ZIVID_MAJOR_VERSION) OR
-   (Zivid_VERSION VERSION_LESS MIN_SUPPORTED_ZIVID_VERSION))
-  MATH(EXPR SUPPORTED_MAJOR_VERSION_PLUS_ONE "${SUPPORTED_ZIVID_MAJOR_VERSION}+1")
-  message(FATAL_ERROR "Zivid version ${Zivid_VERSION} is not supported by this ROS driver! \
-    Zivid ${MIN_SUPPORTED_ZIVID_VERSION} or higher, and less than \
-    ${SUPPORTED_MAJOR_VERSION_PLUS_ONE}.0.0, is required. Hint: Download/install the \
-    newest versions of the Zivid ROS driver and the Zivid SDK.")
-endif()
-
```

Comment:
I see you removed a lot of the error feedback. What kind of error message am I supposed to get now if I try to build with SDK 1.5 installed?
I get the rather cryptic:
`./zivid_camera_settings_generator: symbol lookup error: ./zivid_camera_settings_generator: undefined symbol: _ZN5Zivid10Settings2DD1Ev`
which is difficult for the customer if they don't know that Settings2D was released after 1.5.

Reply:
Never mind, I may not have had a completely clean build. I tried from scratch and got this nice one:

```
  Could not find a configuration file for package "Zivid" that is compatible
  with requested version "1.6.0".

  The following configuration files were considered but not accepted:

    /usr/lib/cmake/Zivid/ZividConfig.cmake, version: 1.5.0

```